app/screens/contributer_screen.rb

```diff
@@ -11,6 +11,7 @@ def on_load
   end
 
   def on_refresh
+    stop_refreshing
```

Comment:
It's just a UI method that tells the UIRefreshControl to stop and for the table to slide back up. Since we're not really doing anything on the `on_refresh` method.